src/Runner.Worker/ExecutionContext.cs

```diff
@@ -36,6 +36,7 @@ public class ExecutionContextType
     public interface IExecutionContext : IRunnerService
     {
         Guid Id { get; }
+        Guid EmbeddedId { get; }
```

Comment:
might be good to update the existing comment further below in this file:

`// Skip if generated context name. Generated context names start with "__". After M271-ish the server will never send an empty context name.`

is wrong, in reality it turned out to be much later

-\> `// Skip if generated context name. Generated context names start with "__". After Enterprise 3.2 the server will never send an empty context name.`